Client/TabManagement/Tab.swift

```diff
@@ -938,6 +939,30 @@ private protocol TabWebViewDelegate: AnyObject {
 }
 
 class TabWebView: WKWebView, MenuHelperInterface {
+    // stubs - these will be given actions in a future task
+    private var accessoryNextAction = { }
+    private var accessoryPreviousAction = { }
+    private var accessoryDoneAction = { }
+    private var accessoryCreditCardAction = { }
```

Comment:
modifiers will need to be accessible from outside as previous -> tab -> credit card helper -> perform task on the webview
```suggestion
    var accessoryNextAction = { }
    var accessoryPreviousAction = { }
    var accessoryDoneAction = { }
    var accessoryCreditCardAction = { }
```

Reply:
@nishant2718
- you should hook up the done button as that is only dismiss 
- where will you be adding stubs to use accessoryNextAction, PreviousAction, CreditCardAction? Is that part of another ticket?